pkg/kubelet/server/stats/summary.go

```diff
@@ -304,10 +301,12 @@ func (sb *summaryBuilder) containerInfoV2ToNetworkStats(info *cadvisorapiv2.Cont
 		txErrors += inter.TxErrors
 	}
 	return &NetworkStats{
-		RxBytes:  &rxBytes,
-		RxErrors: &rxErrors,
-		TxBytes:  &txBytes,
-		TxErrors: &txErrors,
+		Time:      unversioned.NewTime(cstat.Timestamp),
+		StartTime: unversioned.NewTime(info.Spec.CreationTime),
```

Comment:
Well, I think it makes sense in the case of NetworkStats to have it here, since it's the start time of the pod infra container, not actually the start time of the pod (but since we're hiding the infra container implementation, I think the start time of the network metrics makes sense).